ERCS/erc-7558.md

```diff
@@ -0,0 +1,69 @@
+---
+eip: 7558
+title: Minimal Lockable Range NFTs
```

Comment:
Is minimal here describing the standard, or the range? I'd try to reduce the ambiguity in your title.